SynCommons.pas

```diff
@@ -21321,6 +21321,8 @@ function VariantToBoolean(const V: Variant; var Value: Boolean): boolean;
     Value := TVarData(V).VBoolean;
   varInteger: // coming e.g. from GetJsonField()
     Value := TVarData(V).VInteger=1;
+  varString, varUString: // handles "true"/"false" values in JSON

```

Comment:
varUString should be used only with a proper {$ifdef HASVARUSTRING} {$endif} confitional.